tests/utils/pod/pod.go

```diff
@@ -72,3 +75,20 @@ func RedefineWithPVC(pod *corev1.Pod, name string, claim string) *corev1.Pod {
 
 	return pod
 }
+
+func RedefineWithResources(pod *corev1.Pod, limit string, req string) {
+	for i := range pod.Spec.Containers {
+		pod.Spec.Containers[i].Resources = corev1.ResourceRequirements{
+			Limits: corev1.ResourceList{
+				corev1.ResourceCPU: resource.MustParse(limit),
+			},
+			Requests: corev1.ResourceList{
+				corev1.ResourceCPU: resource.MustParse(req),
+			},
+		}
+	}
+}
+
+func RedefineWithRunTimeClass(pod *corev1.Pod, rtc *v1.RuntimeClass) {
```

Comment:
Same here. You only need the (string) rt name.

Reply:
Fixed